### ISSUE TITLE
[List]: The "More" button at the end of the list records section lacks an accessible name

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents-react#6971]
### Describe the bug

<img width="1039" alt="Image" src="https://github.com/user-attachments/assets/1c26cfa1-ae28-46be-a589-68ee510da691" />

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-list--docs

### Reproduction steps

Set the `growing` into `Button`.
Use screen reader and focus on the More button.

### Expected Behaviour

The "More" button at the end of the list records section lacks an accessible name, making it unclear to screen reader users what action it performs.

### UI5 Web Components for React Version

2.7

### UI5 Web Components Version

2.7

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
